esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2016,7 +2019,10 @@ bool CFileSprayEx::onSprayVariable(IEspContext &context, IEspSprayVariable &req,
 
         MemoryBuffer& srcxml = (MemoryBuffer&)req.getSrcxml();
         const char* srcip = req.getSourceIP();
-        const char* srcfile = req.getSourcePath();
+        StringBuffer sourcePath(req.getSourcePath());
+
+        setStandardPosixPath(sourcePath);
```

Comment:
Same comment as above, it might be beneficial to delay the posix conversion until after we test srcxml.len, and srcip and srcfile lines 2026-2032